src/routes/+layout.svelte

```diff
@@ -67,13 +67,8 @@
 						</span>
 					</button>
 					<div class="h-12">
-						<a href="/" class="cursor-pointer" aria-label="Home">
-							<object
-								data="{assets}/favicon.svg"
-								class="h-full"
-								type="image/svg+xml"
-								aria-label=""
-							/>
+						<a href="/" class="cursor-pointer" aria-label="Startseite">
+							<img src="{assets}/favicon.svg" class="h-full" alt="Logo der {config.title}" />
```

Comment:
@mtwente the img tag is not working because of the postbuild (check file history). We either need to use background-url in a div or object tag (or some other none img solution. If it doesn't work at all we need to revert to the inline svg. I didn't want to use the inline solution to load the logo only once...

Reply:
@mtwente let's revert to the inlined svg. check the file history ;)